### DISCUSSION TITLE
Optional fields MongoDB

### DISCUSSION THREAD

**Discussion by SwagiWagi** [prisma/prisma#16613] [Q&A]
I have these models:

```
type Permissions {
///fields
}

model organizations {
    name String
    permissions Permissions?
}
```

My DB:
```
Db.organizations: [
    {
        name: "aName"
    }
]
```

When I call the findMany function, I get the following error:
"\nInvalid `prisma.organizations.findMany()` invocation:\n\n\nAttempted to serialize 'null' with non-composite compatible type '\"Permissions\"' for field permissions."

This is a simplified version of my models.

**Comment by nurul3101** [comment]
Just for reference, this issue is resolved as per our community slack [thread](https://prisma.slack.com/archives/CA491RJH0/p1670235806512779).
@SwagiWagi Would you be able to add the solution which worked for you? This would allow other users who might run into this same issue.

**Reply by SwagiWagi** [reply]
The problem was a nested field in my Permissions type. The error I got from Prisma just didn't reflect it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
